### ISSUE TITLE
checkpoint in where?

### ISSUE THREAD

**Issue by tjshu** [microsoft/DeBERTa#90]
use this command can get checkpoint?and in where?
if i want to save checkpoint,can i add which argument?
python3 -m DeBERTa.apps.run --task_name $task --do_train  \
  --data_dir $cache_dir/glue_tasks/$task \
  --eval_batch_size 128 \
  --predict_batch_size 128 \
  --output_dir $OUTPUT \
  --scale_steps 250 \
  --loss_scale 16384 \
  --accumulative_update 1 \  
  --num_train_epochs 6 \
  --warmup 100 \
  --learning_rate 2e-5 \
  --train_batch_size 32 \
  --max_seq_len 128

thank
